docs/src/emulation.md

```diff
@@ -270,13 +270,13 @@ BrowserContext context = browser.newContext(new Browser.NewContextOptions()
 
 ```python async
 context = await browser.new_context(
-  isMobile=false
+  is_mobile=false
```

Comment:
```suggestion
  is_mobile=False
```

Reply:
Nice catch. I was too focused on the argument name